### ISSUE TITLE
[google_maps_flutter_ios] `-[GoogleMapsUITests testUserInterface]` fails on Xcode 16 beta 4: Failed to get system application, object returned was not an accessibility element: (null)

### ISSUE THREAD

**Issue by jmagman** [flutter/flutter#152970]
```
    t =     5.95s Tap Target Application 'dev.flutter.plugins.googleMobileMapsExample'[0.00, 0.00]
    t =     5.95s     Wait for dev.flutter.plugins.googleMobileMapsExample to idle
    t =     5.95s     Find the Target Application 'dev.flutter.plugins.googleMobileMapsExample'
    t =     5.96s     Check for interrupting elements affecting "google_maps_flutter_example" Application
/Volumes/Work/s/w/ir/x/w/packages/packages/google_maps_flutter/google_maps_flutter_ios/example/ios14/ios/RunnerUITests/GoogleMapsUITests.m:69: error: -[GoogleMapsUITests testUserInterface] : Failed to get system application, object returned was not an accessibility element: (null)
    t =     6.33s Tear Down
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8740319991505781105/+/u/Run_package_tests/native_test/stdout

https://github.com/flutter/packages/pull/7226

**Comment by hellohuanlin** [comment]
FYI beta 5 just got released today: https://xcodereleases.com/

**Comment by jmagman** [comment]
This passed on beta 5 https://github.com/flutter/packages/pull/7226
```
Test Case '-[GoogleMapsUITests testUserInterface]' passed (9.359 seconds).
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8739766869905812641/+/u/Run_package_tests/native_test/stdout

Closing.
